src/Toolkit/Toolkit.Maui/Compass/Compass.cs

```diff
@@ -78,19 +173,79 @@ public bool AutoHide
     /// Identifies the <see cref="GeoView"/> Dependency Property.
     /// </summary>
     public static readonly BindableProperty GeoViewProperty =
-        BindableProperty.Create(nameof(Compass.GeoView), typeof(GeoView), typeof(Compass), null, BindingMode.OneWay, propertyChanged: OnGeoViewPropertyChanged);
+        BindableProperty.Create(nameof(Compass.GeoView), typeof(GeoView), typeof(Compass), null, BindingMode.OneWay, propertyChanged: (s, oldValue, newValue) => ((Compass)s).OnGeoViewPropertyChanged(oldValue as GeoView, newValue as GeoView));
+
+    private void OnGeoViewPropertyChanged(GeoView oldValue, GeoView newValue)
```

Comment:
I spy nullability warnings!
```suggestion
    private void OnGeoViewPropertyChanged(GeoView? oldValue, GeoView? newValue)
```